### DISCUSSION TITLE
Simplified WebRTC session negotiation/setup..

### DISCUSSION THREAD

**Discussion by RussB-tonal** [pion/webrtc#2251] [Q&A]
I'm creating a WebRTC server to support a very simple configuration, as follows:

- The WebRTC connection is only needed to be "half-duplex" (a media stream from the client to the server).

- The WebRTC server has a fixed public IP that the client can connect directly to.  There is no need (thus far) for the server to connect back to the client.

As such, I am thinking that I don't need to use the full "Ice Negotiation" process.  I just need the client to pass the SD for the desired media configuration to the server (and the server to perhaps reply).

Does anyone know of an example or documentation for a simplified connection process?

**Comment by Sean-Der** [comment]
Hi @RussB-tonal 

What version of Pion WebRTC are you using? Do you want to connect with web/other clients?

**Reply by russ-volume** [reply]
I'm using the latest version of Pion Webrtc (recently downloaded from Github).

I only need to support 1 Webrtc video connection from a mobile client to a server instance that has a fixed IP address.

**Reply by Sean-Der** [reply]
Makes sense! I would check out [WHIP](https://www.ietf.org/archive/id/draft-ietf-wish-whip-01.html)

It is standardized signaling for WebRTC ingest. It will reduce the amount of work you need to do on the client side
